### PR TITLE
Cypress/E2E: Fixed channel moderation TM4J keys

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
- Added keys to match TM4J

#### Ticket Link
NA

### CODE REVIEW

**Review by prapti — APPROVED** [review]
These tests look like they are better organized here in Cypress codebase rather than in TM4J where they are kept directly under the "Enterprise" folder. Should those be moved under a "System Console" subfolder too to avoid any less confusion? If so, I can do some of that cleanup next week during my rotation.

**Comment by josephbaylon** [comment]
@prapti I don't mind following the structure of cypress to tm4j for parity

**Comment by prapti** [comment]
Sounds good. I'll clean it up next week :)
Thank you!

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/manage_members_spec.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ describe('MM-23102 - Channel Moderation - Manage Members', () => {
         });
     });
 
-    it('No option to Manage Members for Guests', () => {
+    it('MM-T1547 No option to Manage Members for Guests', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/enterprise/system_console/channel_moderation/manage_members_spec.js:59]: This has 2 steps in TM4J. I think it makes sense to combine the steps there and remain this as is.

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ describe('MM-23102 - Channel Moderation - Post Reactions', () => {
         });
     });
 
-    it('Post Reactions option for Guests', () => {
+    it('MM-T1543 Post Reactions option for Guests', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js:62]: Same here. It has 2 steps in TM4J. I think it makes sense to combine the steps there and remain this as is.

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js [hunk 3]
```diff
@@ -91,7 +91,7 @@ describe('MM-23102 - Channel Moderation - Post Reactions', () => {
         });
     });
 
-    it('Post Reactions option for Members', () => {
+    it('MM-T1544 Post Reactions option for Members', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js:94]: Same here. It has 2 steps in TM4J. I think it makes sense to combine the steps there and remain this as is.

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js [hunk 4]
```diff
@@ -124,7 +124,7 @@ describe('MM-23102 - Channel Moderation - Post Reactions', () => {
         });
     });
 
-    it('Post Reactions option removed for Guests and Members in System Scheme', () => {
+    it('MM-T1545 Post Reactions option removed for Guests and Members in System Scheme', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js:127]: Same here. It has 4 steps in TM4J. I think it makes sense to combine the steps there and remain this as is.

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js [hunk 5]
```diff
@@ -180,7 +180,7 @@ describe('MM-23102 - Channel Moderation - Post Reactions', () => {
 
     // GUEST PERMISSIONS DON'T EXIST ON TEAM OVERRIDE SCHEMES SO GUEST PORTION NOT IMPLEMENTED!
     // ONLY THE MEMBERS PORTION OF THIS TEST IS IMPLEMENTED
-    it('Post Reactions option removed for Guests & Members in Team Override Scheme', () => {
+    it('MM-T1546 Post Reactions option removed for Guests & Members in Team Override Scheme', () => {
```
Comment (saturninoabril) [inline e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js:183]: Could you please check this again? There are 4 steps in TM4J which I could only match step 4 but not the rest of the steps.

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks Joseph. Just one request to check again.

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/manage_members_spec.js (hunk 1 above)
Comment (josephbaylon) [reply e2e/cypress/integration/enterprise/system_console/channel_moderation/manage_members_spec.js:59]: Consolidated steps in TM4J

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js (hunk 2 above)
Comment (josephbaylon) [reply e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js:62]: Consolidated steps in TM4J

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js (hunk 3 above)
Comment (josephbaylon) [reply e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js:94]: Consolidated steps in TM4J

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js (hunk 4 above)
Comment (josephbaylon) [reply e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js:127]: Consolidated steps in TM4J

File: e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js (hunk 5 above)
Comment (josephbaylon) [inline e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js:183]:
```suggestion
    it('MM-T1546_4 Post Reactions option removed for Guests & Members in Team Override Scheme', () => {
```

Comment (josephbaylon) [reply e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js]: Updated key to MM-T1546_4

Comment (saturninoabril) [reply e2e/cypress/integration/enterprise/system_console/channel_moderation/post_reactions_spec.js]: Thanks! Let's deal with steps 1-3 separately.

**Review by saturninoabril — APPROVED** [review]
Thanks @josephbaylon!

**Pull request merged**